### PR TITLE
Social feed etc.

### PR DESCRIPTION
**Pull request by jakubfiala**
This PR:

+ adds the Social Feed functionality which grabs Instagram post embeds dynamically from a proxy API we set up on wellcome.ac.uk.
+ adds interactive pagination to it. Since the pagination works quite differently on mobile/desktop, we simply change element classes with JS, and CSS takes care of the rest.
+ makes some design tweaks inc. breakpoint changes and logo positioning
+ updated the intro video mask SVG so it matches the design and works better altogether

screen shot 2017-09-04 at 10 04 45

### CODE REVIEW

File: components/sections/social-feed/_social-feed.scss [hunk 1]
```diff
@@ -32,19 +32,146 @@
...
-  text-align: center;
+  overflow: hidden;
+  position: relative;
 
   @include respond-to(breakpoint(s)) {
+    @include flex(space-between, stretch, row, nowrap);
     @include rem(margin-bottom, spacing(xl) * 2 + spacing(s));
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 984px;
```
Comment (RhodesPeter) [inline components/sections/social-feed/_social-feed.scss]: Could be inside an @include

File: components/sections/mission/_mission.scss [hunk 2]
```diff
@@ -77,14 +77,18 @@ $instagram-embed-bar-height: 66px;
 }
 
 .mission__boomerang {
-  display: inline-block;
+  display: block;
   margin-bottom: 0;
+  margin-left: auto;
+  margin-right: auto;
   max-height: 100vw;
+  max-width: 400px;
```
Comment (RhodesPeter) [inline components/sections/mission/_mission.scss]: Could be inside an @include

Comment (jakubfiala) [reply components/sections/mission/_mission.scss]: with these max-widths, I'm not rem-ing them, because I don't want them to depend on base font size

Comment (RhodesPeter) [reply components/sections/mission/_mission.scss]: Ok sounds good.

File: components/sections/social-feed/_social-feed.scss [hunk 3]
```diff
@@ -32,19 +32,146 @@
...
+
+    &:last-of-type {
+      @include rem(margin, 0 0 0 spacing(s));
+      @include transform(none);
+      font-size: 0;
+      padding: 0;
+    }
+
+    &:hover {
+      opacity: 1;
```
Comment (RhodesPeter) [inline components/sections/social-feed/_social-feed.scss]: I don't know weather you would find it useful or not but we are using this mixin for opacity: 
```
// Mixin for opacity
@mixin opacity($value, $important: '') {
    @if $important == true {
        $important: ' !important';
    }
    $filter-value: $value * 100;
    opacity: $value #{$important};
    filter: alpha(opacity=$filter-value) #{$important};
}
```

Comment (jakubfiala) [reply components/sections/social-feed/_social-feed.scss]: yep, that's if you need opacity on <IE9, I think we're fine without, but thanks :)

**Pull request merged**